frontend/src/pages/AccountSend.vue

```diff
@@ -469,16 +477,60 @@ function useSendForm() {
       // This does not account for gas fees, but this gets us close enough and we delegate that to the wallet
       await getTokenBalances();
       const { address: tokenAddress, decimals } = token.value;
-      const tokenAmount = parseUnits(humanAmount.value, decimals);
+
+      const currentBalance = balances.value[tokenAddress];
+      const sendingNativeToken = tokenAddress === NATIVE_TOKEN.value.address;
+      let tokenAmount = parseUnits(humanAmount.value, decimals);
+      // Needed for transactions that are going to sendMax.
+      let preCalculatedGasPrice;
+      // Refresh the tokenAmount if the sendMax flag is set.
+      if (sendMax.value) {
+        if (sendingNativeToken) {
+          const [_getGasPrice, _toAddress, _estimatedNativeSendGasLimit] = await Promise.all([
+            provider.value!.getGasPrice(),
+            toAddress(recipientId.value, provider.value!),
+            estimateNativeSendGasLimit(),
+          ]);
+          // Add a 5% buffer to the gasPrice to avoid: "max fee per gas less than block base fee"
+          preCalculatedGasPrice = _getGasPrice.mul('105').div('100');
+          // Get current balance less gas costs.
+          const { ethToSend: balanceLessGasCosts } = await umbraUtils.getEthSweepGasInfo(
+            userAddress.value!,
+            _toAddress,
+            provider.value!,
+            {
+              // We override the gasLimit here because we are sending to an
+              // address that has never been seen before, which increases gas
+              // costs and is not accounted for by getEthSweepGasInfo.
+              gasLimit: _estimatedNativeSendGasLimit,
+              gasPrice: preCalculatedGasPrice,
```

Comment:
Why do we need to fetch and use `preCalculatedGasPrice` instead of just relying on `getEthSweepGasInfo` to do it? (RR since it's a question I guess)

Reply:
Trying to remember, but seems we do this because of the "max fee per gas less than block base fee" error.

But if that's right, the padding we're doing here with `preCalculatedGasPrice` doesn't feel like the correct solution. It seems like the right approach would be to edit `getETHSweepGasInfo` so that it never returns a `gasPrice` less than the expected base fee (if relevant). And then it wouldn't be necessary to pass in a gasPrice to the function as an override.

Reply:
This code was removed. Resolving